xio-core/src/main/java/com/xjeffrose/xio/http/RouteState.java

```diff
@@ -25,4 +26,17 @@ public RouteState(
   public RouteState(RouteConfig config, PipelineRequestHandler handler) {
     this(RouteState::buildRoute, config, handler);
   }
+
+  public static RouteState defaultRoute(PipelineRequestHandler handler) {
+    RouteConfig config = new RouteConfig(ConfigFactory.load().getConfig("xio.defaultRoute"));
+    return new RouteState((c) -> Route.build("*"), config, handler);
```

Comment:
c is not used here. Do we replace this with "ignored"?

Reply:
good call.